cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -161,8 +159,8 @@
   grunt-cli
 ]
 
-default['travis_build_environment']['nvm']['url'] = 'https://raw.githubusercontent.com/creationix/nvm/v0.33.0/nvm.sh'
-default['travis_build_environment']['nvm']['sha256sum'] = '40208b5d10788c257fa4bf7619f4fde57476c75d3e99e17b1cd9b9f413d11a39'
+default['travis_build_environment']['nvm']['url'] = 'https://raw.githubusercontent.com/creationix/nvm/v0.33.2/nvm.sh'
+default['travis_build_environment']['nvm']['sha256sum'] = '77a6da997af26a52fc0647dee849d90d5f36f9222c6d48a5f738672cb850c2d0'
```

Comment:
I can confirm that `77a6da997af26a52fc0647dee849d90d5f36f9222c6d48a5f738672cb850c2d0` is the proper sha256 sum for v0.33.2.